### PR TITLE
Aquatic thyme -- konami code activates /secret

### PR DESCRIPTION
**Pull request by judeallred**

### CODE REVIEW

File: styles/pages/secret.styl [hunk 1]
```diff
@@ -22,11 +22,6 @@
        text-align: center
        &.lit
          background: #FB994A
-  
-  .reset-to-defaults
```
Comment (judeallred) [inline styles/pages/secret.styl]: dead css cleanup

File: src/presenters/footer.jsx [hunk 2]
```diff
@@ -34,6 +38,9 @@ export default function Footer() {
         <img className="for-platforms-icon" src={srcForPlatforms} alt=""/>
         <span className="for-platforms-text">Glitch Teams</span>
       </FooterLine>
+      <Konami>
+        <Redirect to={"/secret"}/>
```
Comment (GregWeil) [inline src/presenters/footer.jsx]: This should use `push={true}` so you can back button out of the secret

Comment (GregWeil) [reply src/presenters/footer.jsx]: Also the footer seems like an odd place for this, I'd suggest bumping it up to at least the layout if not client.jsx

File: src/presenters/includes/konami.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
Comment (pketh) [inline src/presenters/includes/konami.jsx]: I think it  would make sense to call this file `secret.jsx` , to match the `/secret` route and the secret.styl file

**Comment by pketh** [comment]
what does /secret do? I couldn't find it in the diffs

File: src/presenters/includes/konami.jsx (hunk 3 above)
Comment (judeallred) [reply src/presenters/includes/konami.jsx]: not really, since secret.jsx exists?

File: src/presenters/footer.jsx (hunk 2 above)
Comment (judeallred) [reply src/presenters/footer.jsx]: Layout was my other candidate, I'll go for that.

**Comment by judeallred** [comment]
@pketh see https://github.com/FogCreek/Glitch-Community/pull/96

File: src/presenters/pages/secret.jsx [hunk 4]
```diff
@@ -8,9 +8,13 @@ class SecretEffectsOnMount extends React.Component {
   componentDidMount() {    
     //try to play the secret sound:
     const audio = new Audio('https://cdn.glitch.com/a5a035b7-e3db-4b07-910a-b5c3ca9d8e86%2Fsecret.mp3?1535396729988');
-    audio.play().then( () => {
+    const maybePromise = audio.play();
+
+    // Chrome returns a promise which we must handle:
+    maybePromise && maybePromise.then( () => {
```
Comment (GregWeil) [inline src/presenters/pages/secret.jsx]: If you `await audio.play()` instead you can try/catch it instead, which might be a bit cleaner. not a blocker

Comment (judeallred) [reply src/presenters/pages/secret.jsx]: Agreed;   gonna ignore it tho.

**Pull request merged**